tool/python/singa/layer.py

```diff
@@ -228,21 +228,20 @@ def __init__(self, size=0, **kwargs):
 
 class Activation(Layer):
 
-    def __init__(self, activation='stanh', topk=1):
+    def __init__(self, activation='stanh'):
         '''
         required
-          activation = (string)
-        optional
-          topk       = (int)  // the number of results
+          activation = (string) // relu, sigmoid, tanh, stanh, softmax.
         '''
 
+        if activation == 'tanh':
+          print 'Warning: Tanh layer is not supported for CPU'
+
         self.name = activation
-        if activation == 'tanh': activation = 'stanh' # <-- better way to set?
         self.layer_type = enumLayerType(activation)
```

Comment:
Currently, there is a SigmoidLayer, STanhLayer and ReLULayer, hence you can use them if the activation is sigmoid/stanh/relu.
But for tanh, there is no TanhLayer, hence you need to create an ActivationLayer and set its type to TANH. 

A better approach is to uniform all these activation types. I think we can deprecate ReLULayer/SIGMOIDLayer/STanhLayer. But we need to update the ActivationLayer to enable STANH.